### ISSUE TITLE
Can't get th similar accuracy of AVE reported in the paper (81.1%)

### ISSUE THREAD

**Issue by liushenme** [GenjiB/LAVISH#14]
Hi,
We used the same config in this repo to train AVE task on a 3090, but the accuracy we got is 78.96. 

python3 main_trans.py --Adapter_downsample=8 --audio_folder=$PATH/raw_audio --batch_size=2 --early_stop=5 --epochs=50 --is_audio_adapter_p1=1 --is_audio_adapter_p2=1 --is_audio_adapter_p3=0 --is_before_layernorm=1 --is_bn=1 --is_fusion_before=1 --is_gate=1 --is_post_layernorm=1 --is_vit_ln=0 --lr=5e-05 --lr_mlp=4e-06 --mode=train --num_conv_group=2 --num_tokens=2 --num_workers=16 --video_folder=$PATH/video_frames  --is_multimodal=1 --vis_encoder_type=swin

When we use the config in run_v2.sh, the accuracy is 80.05, which is different from those reported in the paper (81.1%). Is the result within the acceptable floating range?

**Comment by GenjiB** [comment]
Hi, 

Can you also use the processed data I provided? I tried on 2~3 servers for v2.sh. It can achieve 80.8~81.1

**Comment by liushenme** [comment]
Yes, we used the processed data you provided in the repo. And the pytorch version we used is 1.13.0.

**Comment by GenjiB** [comment]
can you also try a few other random seeds? I felt like the AVE datasets are too small. It has somewhat issues on reproduce results.

The worse case I got with v2.sh 

![image](https://github.com/GenjiB/LAVISH/assets/32382292/8661cd0f-a584-4553-b5f9-2ccd294b5166)

**Comment by GenjiB** [comment]
hi @kaiw7, 
here are the logs.
[output.log](https://github.com/GenjiB/LAVISH/files/12319724/output.log)
[output_v2.log](https://github.com/GenjiB/LAVISH/files/12319725/output_v2.log)

**Comment by zhenwangrs** [comment]
Hi, @GenjiB , is the accuracy of your paper is based on validation set or test set? Since in your log file I only find the val acc.

**Comment by GenjiB** [comment]
@zhenwangrs  it's the test set. same as here: https://github.com/GenjiB/LAVISH/blob/main/AVE/main_trans_v2.py#L192
